docs(install) Add instructions for switching to kong user on CE (#2462)
Kong/docs.konghq.com: app/_includes/md/ce-kong-user.md

<!-- Shared between all Community Linux installation topics: Amazon Linux,
 CentOS, Debian, RedHat, and Ubuntu -->

>     **Note:** When you start Kong, by default, the Nginx master process will run

Reviewer: ```suggestion
    **Note:** When you start Kong, the Nginx master process runs
```

>     as `root` and the worker processes will run as `nobody`.

Reviewer: ```suggestion
    as `root` and the worker processes run as `nobody` by default.
```

otherwise, it sounds awkward "when you start Kong, by default"

    If this is not the desired behavior, you can switch to the built-in
    `kong` user and group before starting Kong.
    See [Running Kong as a Non-Root User](/latest/kong-user)

>     for more information.

Reviewer: Should always start with For more info, see xyz.
